src/scripts/components/stories/video-js/video-js.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {FunctionComponent, useEffect, useRef} from 'react';
+import videojs, {VideoJsPlayerOptions} from 'video.js';
+import {getStoryAssetUrl} from '../../../libs/get-story-asset-urls';
+
+import {Language} from '../../../types/language';
```

Comment:
I prefer to also indicate with the import that this is a type.
```suggestion
import type {Language} from '../../../types/language';
```

Reply:
![image](https://user-images.githubusercontent.com/45818654/158208426-6b2e6c63-011c-4866-9599-1b6b7318d1d7.png)

probably doesn't work because it's an enum? 🤔

Reply:
Oh ok, than its fine i guess 🤷  :D